src/SpotifyRemote.ts

```diff
@@ -253,20 +252,94 @@ export interface SpotifyRemoteApi extends TypedEventEmitter<SpotifyRemoteEvents>
+    getRecommendedContentItems:
+        nativeModule.getRecommendedContentItems.bind(nativeModule),
+    getRootContentItems: nativeModule.getRootContentItems.bind(nativeModule),
+    isConnectedAsync: nativeModule.isConnectedAsync.bind(nativeModule),
+    pause: nativeModule.pause.bind(nativeModule),
+    playItem: nativeModule.playItem.bind(nativeModule),
+    playItemWithIndex: nativeModule.playItemWithIndex.bind(nativeModule),
+    playUri: nativeModule.playUri.bind(nativeModule),
+    queueUri: nativeModule.queueUri.bind(nativeModule),
+    resume: nativeModule.resume.bind(nativeModule),
+    seek: nativeModule.seek.bind(nativeModule),
+    setRepeatMode: nativeModule.setRepeatMode.bind(nativeModule),
+    setShuffling: nativeModule.setShuffling.bind(nativeModule),
+    skipToNext: nativeModule.skipToNext.bind(nativeModule),
+    skipToPrevious: nativeModule.skipToPrevious.bind(nativeModule),
+    setPlaying: (playing: boolean) => {
+        // todo: Will want to likely check the state of playing somewhere?
+        // Perhaps this can be done in native land so that we don't need to
+        // worry about it here
+        return playing ? SpotifyRemote.resume() : SpotifyRemote.pause();
+    },
+    // Listeners
+    addListener(eventType, listener) {
+        const sub = nativeEventEmitter.addListener(eventType, listener);
+        if (this.listenerCount(eventType) === 0) {
+            nativeModule.eventStartObserving(eventType);
+        }
+        eventListeners.add(sub);
+        return this;
+    },
+    removeListener(eventType, listener) {
+        eventListeners.forEach((eventListener) => {
+            if (
+                eventListener.eventType === eventType &&
+                (!listener || eventListener.caller === listener)
+            ) {
+                eventListener.remove();
+                eventListeners.delete(eventListener);
+            }
+        });
+        if (this.listenerCount(eventType) === 0) {
+            nativeModule.eventStopObserving(eventType);
+        }
+        return this;
+    },
+    removeAllListeners(eventType) {
+        if (eventType) {
+            this.removeListener(eventType);
+            return this;
+        }
+        this.removeListener("playerContextChanged");
+        this.removeListener("playerStateChanged");
+        this.removeListener("remoteConnected");
+        this.removeListener("remoteDisconnected");
+        return this;
+    },
+    emit(eventType, ...args) {
+        nativeEventEmitter.emit(eventType, ...args);
+        return true;
+    },
+    listenerCount(eventType) {
+        let count = 0;
+        eventListeners.forEach((eventListener) => {
+            if (eventListener.eventType === eventType) count += 1;
+        });
+        return count;
+    },
+    on(...args) {
+        return this.addListener(...args);
+    },
+    off(...args) {
+        return this.removeListener(...args);
+    },
+};
+
```

Comment:
Thanks for making this change. Kind of a pain to wrap the underlying native module, but nice to keep the breaking changes to a minimum 👍